tests/python/unittest/test_tvm_testing_features.py

```diff
@@ -210,5 +210,44 @@ def test_pytest_mark_covariant(self, request, target, other_param):
         self.check_marks(request, target)
 
 
+@pytest.mark.skipif(
+    bool(int(os.environ.get("TVM_TEST_DISABLE_CACHE", "0"))),
+    reason="Cannot test cache behavior while caching is disabled",
+)
+class TestCacheableTypes:
```

Comment:
is there any way to assert you're getting a copy in each test case?

Reply:
Good call, and I've added a `TestCachedFixtureIsCopy` to verify that the cached fixture received is a copy.